pandas/tests/frame/test_analytics.py

```diff
@@ -1527,6 +1527,22 @@ def test_duplicated_with_misspelled_column_name(self, subset):
         with pytest.raises(KeyError):
             df.drop_duplicates(subset)
 
+    def test_duplicated_do_not_fail_on_wide_dataframes(self):
```

Comment:
How long does this test take to run? May be worth a slow tag

Reply:
Given that original test case needed to hit recursion limit... it can't be super fast. 
It takes a second or two on my laptop.
I'll add the slow tag